util/Seccomp.c

```diff
@@ -59,6 +60,10 @@ static void catchViolation(int sig, siginfo_t* si, void* threadContext)
 {
     printf("Attempted banned syscall number [%d] see doc/Seccomp.md for more information\n",
            GET_SYSCALL_NUM(si));
+#ifndef si_syscall
```

Comment:
`if (Defined()) {` is a little nicer because it cannot hide compile-time errors.
Also consider removing the `!` marks from the error messages, errors can hit people at the worst times and what seems innocent can be very offensive later on.